docs/pages/setup/guides/fluentd.mdx

```diff
@@ -215,7 +239,7 @@ command.
 $ tctl auth sign --format=tls --user=teleport-event-handler --out=auth
 ```
 
-This command should result in three PEM-encoded files: `auth.crt`, `auth.key`,
+This command should result in three PEM-encoded files, `auth.crt`, `auth.key`,
```

Comment:
```suggestion
This command should result in three PEM-encoded files: `auth.crt`, `auth.key`,
```
I think the colon is clearer since it signals that a list will follow